contracts/RTokenL2.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.19;
+
+import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import { WhitelistAddress } from "./WhitelistAddress.sol";
+
+contract RTokenL2 is ERC20, WhitelistAddress {
+    // --- Constructor ---
+
+    // solhint-disable-next-line no-empty-blocks
+    constructor(string memory name_, string memory symbol_) ERC20(name_, symbol_) { }
```

Comment:
I would name it `R on Base`, `R on Arbitrum`, and so on, and for symbol, maybe `R_BASE`, `R_ARBITRUM`.

Reply:
Hmm, I think it's better to keep the same token ticker like on the mainnet. It's easier to present it as `R Stablecoin`/`R` only and when we show the cumulative supply across chains, we show it under `R`, not separately.

Reply:
it is fine anyway, we parametrise it in constructor